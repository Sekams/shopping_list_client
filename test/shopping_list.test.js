import React from 'react';
import ShoppingList from '../src/components/shopping_list'
import MockRouter from 'react-mock-router';

test("renders the shopping_list component", () => {
    const wrapper = shallow(
        <MockRouter>
            <ShoppingList />
        </MockRouter>
    );
    expect(wrapper).toMatchSnapshot();
});