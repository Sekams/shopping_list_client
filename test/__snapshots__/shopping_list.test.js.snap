// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the shopping_list component 1`] = `<ShoppingList />`;
